### PR TITLE
Archives header

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Style updates for the archive page template.

**Motivation and Context**
Moves the pagination for archive pages into the header area.

**How Has This Been Tested?**
Changes are in dev.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: template-parts/header_content-archive.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Header content template for the 'Section' page template
+ */
+?>
+
+<?php
+global $post;
+
+$title_string = wptexturize( ucfwp_get_header_title( null ) );
```
Comment (cjg89) [inline template-parts/header_content-archive.php]: `ucfwp_get_header_title()` returns a texturized title--no need to call `wptexturize()` here

File: template-parts/header_content-archive.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Header content template for the 'Section' page template
+ */
+?>
+
+<?php
+global $post;
```
Comment (cjg89) [inline template-parts/header_content-archive.php]: `global $post;` can be removed--it's not referenced anywhere, and I don't think it'll actually be set in archive views anyway

File: template-parts/header_content-archive.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+$title_string = wptexturize( ucfwp_get_header_title( null ) );
+
+ob_start();
+
+?>
+<h1 class="self-align-middle text-uppercase h4 mb-0"><?php echo $title_string; ?></h1>
```
Comment (cjg89) [inline template-parts/header_content-archive.php]: `self-align-middle`: did you mean `align-self-middle`?

File: archive.php [hunk 4]
```diff
@@ -5,7 +5,6 @@
 <div class="container">
```
Comment (cjg89) [inline archive.php]: Consider adding `mb-5` to this container to increase the amount of space between the main content and footer, particularly on smaller screen sizes

File: archive.php [hunk 5]
```diff
@@ -5,7 +5,6 @@
...
 	<div class="row">
 		<div class="col-md-8">
```
Comment (cjg89) [inline archive.php]: `col-lg-8` would be ideal here for consistency with other two column layouts in the new theme; the -md breakpoint starts to get a bit scrunched

File: archive.php [hunk 6]
```diff
@@ -5,7 +5,6 @@
...
-			<?php today_archive_pagination(); ?>
 			<?php if ( have_posts() ) : ?>
 				<?php echo today_post_list_display_feature( '', $posts, array( 'posts_per_row' => 3, 'layout' => 'vertical' ) ); ?>
 				<?php ucfwp_the_posts_pagination(); ?>
```
Comment (cjg89) [inline archive.php]: `col-lg-4 pl-lg-5` would be ideal over `col-md-4` for the sidebar wrapper column.  Adding `mt-4 mt-lg-0` would also improve vertical spacing between the main content and collapsed sidebar at smaller screen sizes

File: template-parts/header_content-archive.php (hunk 3 above)
Comment (jmbarne3) [reply template-parts/header_content-archive.php]: Removed the class. That was leftover from me trying to figure out how to place layout things properly.

File: template-parts/header_content-archive.php (hunk 1 above)
Comment (jmbarne3) [reply template-parts/header_content-archive.php]: Removed.

File: template-parts/header_content-archive.php (hunk 2 above)
Comment (jmbarne3) [reply template-parts/header_content-archive.php]: Removed.

File: archive.php (hunk 5 above)
Comment (jmbarne3) [reply archive.php]: Updated.

File: archive.php (hunk 6 above)
Comment (jmbarne3) [reply archive.php]: Also updated.

File: archive.php (hunk 4 above)
Comment (jmbarne3) [reply archive.php]: Done

**Pull request merged**